cypher/cypher-docs/src/docs/dev/ql/administration/security/show-user-privileges-syntax.asciidoc

```diff
@@ -1,7 +1,7 @@
 .Command syntax
 [source, cypher]
 -----
-SHOW USER[S] [user1[, user2]] PRIVILEGE[S] [AS [REVOKE] COMMAND[S]]
+SHOW USER[S] [user[, ...]] PRIVILEGE[S] [AS [REVOKE] COMMAND[S]]
```

Comment:
For consistency with the syntax below:
```suggestion
SHOW USER[S] [name[, ...]] PRIVILEGE[S] [AS [REVOKE] COMMAND[S]]
```

Reply:
The `SHOW ROLE[S] role[, ...] PRIVILEGE[S]` has `role` and not `name` so I think I want to keep `user` here. 

I can add a card on our wall to look over where we have `name` and where we have something else and make it more consistent/clear between different commands, within commands and different places for the same command :)

Reply:
To make this particular command consistent we should change in the user/role management part to have `user`/`role` instead of `name` (since the [new list privilege section](https://github.com/neo4j/neo4j-documentation/pull/753) adds additional places with `user`/`role`)